providers/tests/google/cloud/operators/test_datacatalog.py

```diff
@@ -102,9 +102,20 @@
     "name": TEST_ENTRY_PATH,
 }
 TEST_ENTRY_GROUP: EntryGroup = EntryGroup(name=TEST_ENTRY_GROUP_PATH)
-TEST_ENTRY_GROUP_DICT: dict = {"description": "", "display_name": "", "name": TEST_ENTRY_GROUP_PATH}
+TEST_ENTRY_GROUP_DICT: dict = {
+    "description": "",
+    "display_name": "",
+    "name": TEST_ENTRY_GROUP_PATH,
+    "transferred_to_dataplex": False,
+}
 TEST_TAG: Tag = Tag(name=TEST_TAG_PATH)
-TEST_TAG_DICT: dict = {"fields": {}, "name": TEST_TAG_PATH, "template": "", "template_display_name": ""}
+TEST_TAG_DICT: dict = {
```

Comment:
@gopidesupavan I suggest doing similar to https://github.com/apache/airflow/pull/44037/files

Reply:
ah i see, got it. i will update that. thank you